web2/htdocs/index.html

```diff
@@ -167,7 +167,32 @@ <h2>You Are About To Restore Data</h2>
               <button class="closes danger" rel="yes">Yes, Initiate Restore</button>
             </div>
           </div>
+          
     <!-- }}} --></script>
+
+    <script type="text/html" id="template:delete-are-you-sure"><!-- {{{ -->
+      [[#
+        {delete-are-you-sure}
+
+        A modal interaction screen that requires the operator to acknowledge
+        that what they are about to do is in fact to delete data, that it
+        might be dangerous, and it might not work out.
+      ]]
+          <div class="confirm">
+            <h2>You Are About To Delete Your Backup Job</h2>
```

Comment:
I would like some additional information about this job.  How many archives are attached to it?  Do those get deleted / purged when I remove the job?  When does this job normally run?  What is it's schedule? etc.

Provide the operator with enough information to make them 100% sure they are deleting the right thing, and didn't mis-click.